src/site/shared/containers/HistoryBox/index.tsx

```diff
@@ -71,19 +71,26 @@ export const HistoryBox = ({ info }: Props) => {
     const dispatch = useSharedDispatch();
 
     const { undoHistory, redoHistory } = useHistory(info);
-
     return (
         <div className={`historybox ${isOpen ? "" : "historybox__move"} ${isHistoryBoxOpen ? "" : "hide"}`}>
             <div>
                 <span>History</span>
                 <span onClick={() => dispatch(CloseHistoryBox())}>×</span>
             </div>
             <div>
+                { redoHistory.length > 0 ? (<>
+                    {[...redoHistory].reverse().map((a, i) =>
+                        <HistoryEntry key={`history-box-dashedentry-${i}`} a={a} isRedo></HistoryEntry>
+                    )}
+                    <div style={{textAlign: 'center', fontWeight: 'bold'}}> Redo </div>
+                    <div className={"historybox__separator"} > </div>
+                 </>) : (<></>)}
```

Comment:
You can go even crazier with this and instead of doing a ternary operator where the false case is just empty, you can replace it with `&&`. Broadly, it would look like `(condition) && (html)`. Here's an example of it in place: https://github.com/OpenCircuits/OpenCircuits/blob/2746d42f1d75d5097514c08b54dd449641c3865e/src/site/shared/containers/Header/Right/index.tsx#L33-L35